xbmc/network/EventServer.cpp

```diff
@@ -186,7 +182,7 @@ void CEventServer::Run()
     CLog::Log(LOGERROR, "ES: Invalid port range specified %d, defaulting to 10", port_range);
     port_range = 10;
   }
-  if (!m_pSocket->Bind(any_addr, m_iPort, port_range))
+  if (!m_pSocket->Bind(!CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_ESALLINTERFACES), m_iPort, port_range))
```

Comment:
But is there a real use case for it? you obviously have a modified setup with other apps running locally. Regular users might not configure their firewall but do they make use of the local bind? Seems more power user oriented.